test/libkqueue.c

```diff
@@ -20,9 +20,10 @@ static void
 test_libkqueue_version(struct test_context *ctx)
 {
     struct kevent kev, receipt;
+    struct timespec ts = {0, 0};
 
     EV_SET(&kev, 0, EVFILT_LIBKQUEUE, EV_ADD, NOTE_VERSION, 0, NULL);
-    if (kevent(ctx->kqfd, &kev, 1, &receipt, 1, &(struct timespec){}) != 1) {
+    if (kevent(ctx->kqfd, &kev, 1, &receipt, 1, &ts) != 1) {
```

Comment:
Windows still doesn't support C11?!

Reply:
I think it still doesn't support compound literals. I even tried enabling the latest conforming preprocessor:

https://devblogs.microsoft.com/cppblog/announcing-full-support-for-a-c-c-conformant-preprocessor-in-msvc/